imports.go

```diff
@@ -30,22 +31,21 @@ func parseImports(recursive bool, testDependencies bool) ([]string, error) {
 
 	// Ensuring our dependencies exists isn't a strict requirement, therefore
 	// only print a message to stderr rather then completely failing.
-	for _, pkg := range packages {
-		err = ensureDependenciesExist(pkg, testDependencies)
-		if err != nil {
-			logger.Warning(err)
-		}
+
+	err = ensureDependenciesExist(packages, true)
+
```

Comment:
can you please remove this newline between call func and handling error? 

manul is written with code style like
```
err := fn()
if err != nil {
  // handle error
}
```
without newline between calling func and handling error

Reply:
I think I've removed all the stray line feeds now.  Best check.